timbuctoo-instancev4/src/spec/java/nl/knaw/huygens/timbuctoo/v5/graphql/datafetchers/LookupFetcherWrapperTest.java

```diff
@@ -0,0 +1,141 @@
+package nl.knaw.huygens.timbuctoo.v5.graphql.datafetchers;
+
+import graphql.execution.ExecutionId;
+import graphql.language.Field;
+import graphql.language.FragmentDefinition;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.DataFetchingFieldSelectionSet;
+import graphql.schema.GraphQLOutputType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.GraphQLType;
+import nl.knaw.huygens.timbuctoo.v5.graphql.datafetchers.dto.SubjectReference;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class LookupFetcherWrapperTest {
+
+  @Test
+  public void handlesAbsoluteUrls() {
+    Mock lookupFetcherMock = new Mock();
```

Comment:
I think the name mock is not very describing. I think the name LookUpFetcherMock would be better.